src/util/precisionUtil.ml

```diff
@@ -1,23 +1,31 @@
 (* We define precision by the number of IntDomains activated.
  * We currently have 4 types: DefExc, Interval, Enums, Congruence *)
-type precision = (bool * bool * bool * bool)
+type int_precision = (bool * bool * bool * bool)
+(* Same applies for FloatDomain
+ * We currently have only an interval type analysis *)
```

Comment:
I assume this is to allow us (or others for that matter) to add further types of float analyses? I think especially a DefExc would be very helpful for excluding invalid values. Good Job for thinking ahead